lib/components/narrative/default/default-itinerary.js

```diff
@@ -1,34 +1,76 @@
+import moment from 'moment-timezone'
 import coreUtils from '@opentripplanner/core-utils'
 import React from 'react'
+import { FormattedMessage, FormattedNumber } from 'react-intl'
+import { connect } from 'react-redux'
 
 import NarrativeItinerary from '../narrative-itinerary'
 import ItineraryBody from '../line-itin/connected-itinerary-body'
-import ItinerarySummary from './itinerary-summary'
 import SimpleRealtimeAnnotation from '../simple-realtime-annotation'
-import { getTotalFareAsString } from '../../../util/state'
 
-const { isBicycle, isTransit } = coreUtils.itinerary
-const { formatDuration, formatTime } = coreUtils.time
+import ItinerarySummary from './itinerary-summary'
+
+const { calculateFares, isBicycle, isTransit, isMicromobility } = coreUtils.itinerary
 
+/**
+ * Obtains the description of an itinerary in the given locale.
+ */
 // FIXME move to core utils
```

Comment:
I think this is specific enough especially now that it is using the `FormattedMessage` component that it should not be moved to the `@opentripplanner/core-utils` package.  However, it definitely seems like it could be moved into a util file within the otp-rr project.